### PR TITLE
WIP removing extractorapp

### PR DESCRIPTION
**Pull request by emmdurin**
Depends on [PR#3815](https://github.com/georchestra/georchestra/pull/3815)

### CODE REVIEW

File: ldap/README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ The basic users:
  * ```testeditor``` has the USER & GN_EDITOR roles. The password is **testeditor**.
  * ```testadmin``` has the USER, GN_ADMIN, ADMINISTRATOR and MOD_* roles. The password is **testadmin**.
  * ```testdelegatedadmin``` has the USER role. Is able to grant the EXTRACTORAPP & GN_EDITOR roles to members of the psc & c2c orgs. The password is **testdelegatedadmin**.
- * ```geoserver_privileged_user``` is a required user. It is internally used by the extractorapp, mapfishapp & geofence modules. The default password is ```gerlsSnFd6SmM``` (you should change it, and update the datadir as explained in its [README](https://github.com/georchestra/datadir/blob/18.06/README.md)).
+ * ```geoserver_privileged_user``` is a required user. It is internally used by the extractorapp & geofence modules. The default password is ```gerlsSnFd6SmM``` (you should change it, and update the datadir as explained in its [README](https://github.com/georchestra/datadir/blob/18.06/README.md)).
```
Comment (pmauduit) [inline ldap/README.md:40]: extractorapp still mentioned ?

File: ldap/docker-root/georchestra.ldif [hunk 2]
```diff
@@ -96,7 +96,7 @@ sn: geoserver_privileged_user
 mail: psc+geoserver_privileged_user@georchestra.org
 uid: geoserver_privileged_user
 cn: geoserver_privileged_user
-description: Do not modify.  This is a required user for extractorapp, mapfishapp...
+description: Do not modify.  This is a required user for extractorapp, ...
```
Comment (pmauduit) [inline ldap/docker-root/georchestra.ldif:99]: same comment, maybe we shall rephrase it like "this is a required user for server-to-server internal communications"

**Review by pmauduit — COMMENTED** [review]
I am surprised that the extractorapp/ directory removal does not appear in the diff, is it because the other PR about removing mapfishapp is pending (and the reason why this one is in a draft state) ?

**Comment by emmdurin** [comment]
> I am surprised that the extractorapp/ directory removal does not appear in the diff
Because it is just not done.

**Pull request merged**